src/cs2_demo_downloader/match_scraper.py

```diff
@@ -72,7 +85,7 @@ def goto_personal_data(page, url):
 
     if "Personal Game Data" not in page.title():
         logging.error("Can't get to Personal Game Data page")
-        authenticate(True)
+        authenticate(None, None, True)
```

Comment:
Probably worth writing a comment about why you're passing `None` for username and password here, or otherwise just having a think about how the `authenticate` function works to see if there isn't a better way to get the result you want from it.

If it can do two things and one of those things **requires** you to pass `None`, maybe that use case should be its own method.

Reply:
Good idea made it a bit more clear.